lib/tufts/curation/vocab/tufts.rb

```diff
@@ -8,7 +8,6 @@ module Vocab
       # http://dl.tufts.edu/terms/identifier
       class Tufts < RDF::Vocabulary('http://dl.tufts.edu/terms#')
         # term :VotingRecord
-        # term :EAD
```

Comment:
There's no data you need to clear out of this field before you make it inaccessible?

Reply:
Ah, you're right, it might be handy for the data remediation phase.  I'll put it back.

Reply:
What does it mean that it was commented out?